README.md

```diff
@@ -24,18 +24,34 @@ Install Node modules:
 npm install
 ```
 
-Install the docker container with the database:
+Start the docker containers in the background:
 ```
-docker run --name rra-postgis \
--e "POSTGRES_PASSWORD=rra" \
--e "POSTGRES_USER=rra" \
--e "POSTGRES_DB=rra" \
--p 5432:5432 \
--d \
-mdillon/postgis
+docker-compose up -d
 ```
+Stop the docker containers with:
+```
+docker-compose stop
+```
+
+The containers will store the information within themselves. If the container is deleted all the information will be lost.
+When using the application locally [Minio](https://minio.io/) is used as cloud storage to simulate S3. It's interface will be available at `http://localhost:9000`.
+
+### Setup
+Both the database and the local storage need some setup. Before running the setup add the appropriate values to the config files. (See section below)
+```
+npm run setup
+```
+Will prepare all the needed tables for the database and the bucket for storage. Both the database and the bucket will be removed and created anew.
 
-Once the container is installed use `docker stop rra-postgis` to stop it and `docker start rra-postgis` to start it again.
+Other available options:
+- `--db` - Sets up database structure
+- `--data` - Sets up database and data fixtures
+- `--s3` - Sets up the storage bucket.
```

Comment:
Minio is pretty much an `s3` proxy. Their client can be used to interact with s3, but it's a valid point. Will do